src/node_mgmt/commands.py

```diff
@@ -211,6 +212,61 @@ def readall(s):
     return result
 
 
+def holykell_sensor_address(node):
+    from time import sleep
+    import minimalmodbus
+
+    mod = minimalmodbus.Instrument('/dev/ttyAMA0', 1, debug=True)
```

Comment:
Maybe set some of these as constants, like in https://github.com/ammpio/ammp-edge/blob/main/src/node_mgmt/env_scan.py#L20, so it's more obviously what the meanings of the parameters are